runner/ansible/roles/1.4.2/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '1.4.2'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    cf_status=$(crm_attribute -t crm_config -G -n concurrent-fencing --quiet)
+    if cibadmin -Q --xpath "//primitive[@type='fence_azure_arm']/@type" > /dev/null 2>&1; then
+        if [[ ${cf_status} = "true" ]]; then
+            echo "correct"
```

Comment:
I would prefer to use return codes here.
The `correct` word is just a hardcoded value, that doesn't add any value.
In this case, this would be a correct return code, the next `else` would be an error one, and the 3rd one a correct one again

Reply:
To my mind, it can as well be the "correct" string as long as it's consistent. We use however different string literals in the checks, so maybe, it'll be easier to implement with numeric codes.

Reply:
I can go with a return code here.

Reply:
Have moved to an RC for this one.